vector/src/main/res/layout/activity_home.xml

```diff
@@ -30,74 +34,130 @@
-            </FrameLayout>
-        </RelativeLayout>
+                android:tint="@android:color/white"
+                app:fab_size="mini"
+                app:fab_colorNormal="@color/vector_fuchsia_color"
+                app:fab_colorPressed="@color/vector_fuchsia_color"
+                app:fab_icon="@drawable/ic_person_black_24dp"
+                app:fab_title="@string/room_recents_start_chat"/>
+
+            <com.getbase.floatingactionbutton.FloatingActionButton
+                android:id="@+id/button_create_room"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                app:fab_size="mini"
+                app:fab_colorNormal="@color/vector_fuchsia_color"
+                app:fab_colorPressed="@color/vector_fuchsia_color"
+                app:fab_icon="@drawable/ic_add_white"
+                app:fab_title="@string/room_recents_create_room"/>
+
+            <com.getbase.floatingactionbutton.FloatingActionButton
+                android:id="@+id/button_join_room"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                app:fab_size="mini"
+                app:fab_colorNormal="@color/vector_fuchsia_color"
+                app:fab_colorPressed="@color/vector_fuchsia_color"
+                app:fab_icon="@drawable/riot_tab_rooms"
+                app:fab_title="@string/room_recents_join_room"/>
+
+        </com.getbase.floatingactionbutton.FloatingActionsMenu>
 
         <android.support.design.widget.BottomNavigationView
             android:id="@+id/bottom_navigation"
-            android:layout_width="match_parent"
+            android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_alignParentBottom="true"
             android:background="?attr/riot_primary_background_color"
             app:itemBackground="?attr/riot_primary_background_color"
             app:itemIconTint="@color/home_bottom_nav_view_tint"
             app:itemTextColor="@color/home_bottom_nav_view_tint"
-            app:menu="@menu/bottom_navigation_main" />
-    </RelativeLayout>
+            app:menu="@menu/bottom_navigation_main"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintBottom_toBottomOf="parent" />
+
+        <!-- Elevate above BottomNavigationView (above lollipop). -->
+        <View
+            android:id="@+id/floating_action_menu_touch_guard"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintBottom_toBottomOf="parent"
+            android:clickable="false"
+            android:elevation="10dp"
```

Comment:
The touch guard does not work on Pre-Lollipop which is a bit annoying. Maybe the touch guard can be placed before in the XML file to be displayed under the FAB menu